public/js/components/subscription.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component, PropTypes } from 'react';
+
+import { gettext } from 'utils';
+import * as products from 'products';
+
+
+export default class Subscription extends Component {
+
+  static propTypes = {
+    seller_product: PropTypes.shape({
+      public_id: PropTypes.string,
+    }).isRequired,
+  }
+
+  render() {
+    var productData = products.get(this.props.seller_product.public_id);
+
+    return (
+      <div className="subscription">
+        <div className="product">
+          <img src={productData.img}/>
+          <h2 className="seller">{productData.seller.name.en}</h2>
+          <div className="description">{productData.description.en}</div>
+        </div>
+        <div className="price">
+          <div>{productData.price.en}</div>
+          <div>/ {gettext('month')}</div>
+        </div>
+        <div className="clear"></div>
+        <div className="subnav">
+          <div className="change-pay-account">
```

Comment:
the pain! make it stop! :fire: I tried styling the a tags as blocks and just couldn't get it to work.